packages/web/src/pages/upload-page/pages/FinishPage.tsx

```diff
@@ -170,13 +158,15 @@ export const FinishPage = (props: FinishPageProps) => {
         return upload.tracks?.[0].metadata.permalink
       case UploadType.ALBUM:
       case UploadType.PLAYLIST:
-        return collectionPage(
-          user!.handle,
-          upload.metadata?.playlist_name,
-          upload.completionId,
-          null,
-          uploadType === UploadType.ALBUM
-        )
+        return upload.completedEntity
+          ? collectionPage(
+              null,
+              null,
+              null,
```

Comment:
maybe just make this take an object?

Reply:
Too many references, I think we'll save that for later :joy:

I think the real solution is to only take in permalinks